README.md

```diff
@@ -1,79 +1,55 @@
 # kind-local
 
-This is a Proof of Concept of how to run a local container registry in [Kind](https://github.com/kubernetes-sigs/kind) (Kubernetes in Docker).
+This repo documents the best way to run a local [Kind](https://github.com/kubernetes-sigs/kind) cluster with Tilt.
 
 ## Why
 
 When developing locally, you want to push images to the cluster as fast as possible.
 
-Pushing to an in-cluster container registry skips a lot of overhead:
+Pushing to an in-cluster image registry skips a lot of overhead:
 
 - Unlike with a remote registry, the image stays local to your machine, with no network traffic
 
 - Unlike with `kind load`, docker will skip pushing any layers that already exist in the registry
 
 This makes it a great solution for iterative local development. But setting it up is awkward and fiddly.
 
-This repo demonstrates how you might build a more robust solution.
+This repo has scripts for creating a Kind cluster, and applying labels to it so that Tilt
```

Comment:
"This repo has scripts for creating a Kind cluster with its own local image registry, and applying labels to it so that Tilt can find the registry."

Reply:
felt redundant after changes above, so i just axed this